utils.ts

```diff
@@ -63,3 +64,56 @@ export function mapToChannels(channelNames: string[], guild: Guild): TextChannel
     .filter(channel => channel)
     .filter(channel => !_.includes(blacklisted, channel.name.toLowerCase())) as TextChannel[];
 }
+
+export async function initializeBotAudit(bot: CommandoClient)
+{
+  try {
+    if (process.env.AUDIT_CHANNEL) {
+      const auditChannel = process.env.AUDIT_CHANNEL;
+      var guild = bot.guilds.cache.first();
+      var alreadyMade = guild.channels.cache
+        .filter(channel => channel.isText())
+        .map(channel => channel.name.toLowerCase())
+        .includes(auditChannel);
+
+      if (!alreadyMade) {
+        var role = await createChannel(bot, auditChannel, guild);
+        // add the new role to each mod.
+        guild.roles.cache
+          .filter(role => role.name.toLocaleLowerCase() == process.env.MOD_ROLE.toLowerCase())
+          .flatMap(role => role.members)
+          .forEach(member => {
+            member.roles.add(role);
+          });
+      }
```

Comment:
As we discussed in chat, I don't believe this channel should have its own role. It should use the @moderator roles for its permissions.